src/redux/data/todos.js

```diff
@@ -0,0 +1,6 @@
+import { keyValue } from 'trampss-redux-factory'
+import { mapPayload } from 'trampss-redux-factory/helpers'
+
+/* Todos Reducer  with her mapper */
+const todosMapper = todo => ({ id: new Date().getTime(), todo })
+export default keyValue({ pre: [mapPayload(/@trampss\/ADD_TODOS/)(todosMapper)] })({ path: 'data', key: 'id', name: 'todos' })
```

Comment:
You don't use the exported path from data.js

Reply:
👍